src/server/bootstrap.rs

```diff
@@ -88,7 +95,7 @@ async fn get_server_status(server_directory: &Path) -> crate::Result<ServerStatu
 async fn initialize_server(
     gsettings: &GlobalSettings,
     server_cfg: ServerConfig,
-) -> anyhow::Result<(impl Future<Output = anyhow::Result<()>>, Rc<Notify>)> {
+) -> anyhow::Result<(impl Future<Output=anyhow::Result<()>>, Rc<Notify>)> {
```

Comment:
It seems that your local formatter is overriding the formatter that we use. What Rust version do you use, 1.55?
If this formatting is produced by IntelliJ, please enable `Use rustfmt instead of built-in formatter` in `Settings -> Languages & Frameworks -> Rust -> Rustfmt`.